uber/site_sections/registration.py

```diff
@@ -24,11 +16,21 @@ def check_everything(attendee):
     if message:
         return message
 
-    if AT_THE_CON and not attendee.age_group and attendee.is_new:
+    if c.AT_THE_CON and not attendee.age_group and attendee.is_new:
         return "You must enter this attendee's age group"
 
 
-@all_renderable(PEOPLE, REG_AT_CON)
+def check_atd(func):
```

Comment:
Regarding this, one of my goals for this year was to unify the at-door and pre-reg pages. I can go into details on why if you want, but basically the pre-reg workflow is better for attendees and having just one page to worry about makes life a lot easier when it comes to configuring pages, theming, and testing bugs.

This is actually partially-done in kickin_fixes branch if you ever want to have a look.